datasets/ru/acf_bribetakers/crawler.py

```diff
@@ -51,7 +42,9 @@ def parse_result(context: Context, row: Dict[str, Any]):
     for tl in transliterations.split("\n"):
         tl = h.remove_bracketed(tl)
         entity.add("alias", tl)
-    entity.add("birthDate", parse_date(dob), original_value=dob)
+    if dob and NO_YEAR.match(dob):
+        dob = None
+    entity.add("birthDate", h.extract_date(context.dataset, dob), original_value=dob)
```

Comment:
I decided to keep `entity.add` instead of `h.apply_date` not to lose the `original_value`